### PR TITLE
fix: `NullableTypeDeclarationFixer` - don't convert standalone `NULL` into nullable union type

### PR DESCRIPTION
**Pull request by kubawerlos**
Follow up to https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8098

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -198,7 +198,7 @@ jobs:
         env:
           PHP_CS_FIXER_IGNORE_ENV: ${{ matrix.PHP_CS_FIXER_IGNORE_ENV }}
           FAST_LINT_TEST_CASES: ${{ matrix.FAST_LINT_TEST_CASES }}
-        run: vendor/bin/paraunit run --testsuite unit,integration
+        run: vendor/bin/phpunit tests/Fixer/LanguageConstruct/NullableTypeDeclarationFixerTest.php
```
Comment (keradus) [inline .github/workflows/ci.yml:201]: i guess we should rollback before merge

Comment (kubawerlos) [reply .github/workflows/ci.yml:201]: As always - with the fix, but let's fix SCA issue first.

**Comment by Wirone** [comment]
Thanks @kubawerlos 🍻!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (keradus) [reply .github/workflows/ci.yml]: then I suggest to keep PR as draft while there are hidden todo in code,
i got impression that PR is prepared and ready for review on moment I was reviewing it, and I found this gem

Comment (kubawerlos) [reply .github/workflows/ci.yml]: PR is not ready for review when CI is failing.

The switching from Paraunit to PHPUnit is to have a diff in output, which Paraunit unfortunately does not do.

Comment (keradus) [reply .github/workflows/ci.yml]: way easier for me to determine it's not ready by seeing "WIP" or "DRAFT" indicator, than checking the CI to check what exactly failed and guess if it was intended or not. 

Please, consider to indicate as drafts, to make this explicit

Comment (Wirone) [reply .github/workflows/ci.yml]: We really need facile-it/paraunit#215 get fixed or we need to consider different solutions. It was already discussed in #7187 and I agree that current ParaUnit's output is hard to work with: not only it does not provide diffs, but also prints too many superfluous information that makes debugging failures much harder. Maybe we can consider dropping 7.4 support, that would also bump our minimal PHPUnit version and maybe ParaTest can be implemented instead? I don't know, but we need to fix it somehow.

Comment (Wirone) [reply .github/workflows/ci.yml]: Side note: personally I consider PR as reviewable when it's _not_ marked as a draft (unless author asks for it), and when it comes to CI failures I believe they should not stop the review as failures can be accidental. For bug fixes I am used to temporary changes during reproduction period, so I just ignore them and I just review added test cases, then when fix is provided I verify the changes in the source code and approve when everything is correct. I don't need the feel to mark PR as draft only for few minutes needed to make CI failing and to push the fix (which I assume is already done, but pushed separately as we encourage to do). Just my 2 cents.

**Pull request merged**